doc/releases/release-notes-3.0.rst

```diff
@@ -80,6 +80,13 @@ Removed APIs in this release
 * Removed ``flash_write_protection_set()`` along with the flash write protection
   implementation handler.
 
+* Removed :c:macro:`CAN_BUS_UNKNOWN` and changed the signature of
+  :c:func:`can_get_state` to return an error code instead.
+
+* Removed :c:macro:`DT_CHOSEN_ZEPHYR_CANBUS_LABEL` in favor of utilizing
```

Comment:
nit: use literals for removed macros/funcs/etc

```suggestion
* Removed ``CAN_BUS_UNKNOWN`` and changed the signature of
  :c:func:`can_get_state` to return an error code instead.

* Removed ``DT_CHOSEN_ZEPHYR_CANBUS_LABEL`` in favor of utilizing
```

Reply:
@gmarull Thanks, fixed.